runtime/caml/lf_skiplist.h

```diff
@@ -0,0 +1,128 @@
+/**************************************************************************/
+/*                                                                        */
+/*                                 OCaml                                  */
+/*                                                                        */
+/*               Sadiq Jaffer, OCaml Labs Consultancy Ltd                 */
+/*               Xavier Leroy, projet Cambium, INRIA Paris                */
+/*                                                                        */
+/*   Copyright 2021 OCaml Labs Consultancy Ltd                            */
+/*   Copyright 2020 Institut National de Recherche en Informatique et     */
+/*     en Automatique.                                                    */
+/*                                                                        */
+/*   All rights reserved.  This file is distributed under the terms of    */
+/*   the GNU Lesser General Public License version 2.1, with the          */
+/*   special exception on linking described in the file LICENSE.          */
+/*                                                                        */
+/**************************************************************************/
+
+/* A concurrent dictionary data structure implemented as skip lists. See
+   implementation for much more detail. */
+
+/* Keys and associated data are natural-width integers (type [uintnat]).
+   Key values 0 and uintnat_max are reserved for internal use. Do not use.
```

Comment:
```suggestion
   Key values 0 and uintnat_max are reserved for internal use; do not use.
```

just a minor tweak to connect the "do not use" to the key values sentence.